chall01/phakakos.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import string
+import re
+
+pattern = re.compile("^[a-zA-Z ]+$")
+
+def pUsage():
+        print("usage: ./phakakos.py <a-zA-Z string>")
```

Comment:
You should prefer using `sys.argv[0]` to display the program name, it's a good habit.

Reply:
Never thought about that, so now I know to use it in my bash scripts too.